source/refined-github.css

```diff
@@ -186,3 +186,9 @@ pr-branches
 .TimelineItem[id^='event-'] .TimelineItem-body :is(ins, del) {
 	display: block !important;
 }
+
+/* Resize large images for blob preview */
+div[data-target='readme-toc.content'] div.blob-wrapper img {
+	max-width: 100%;
+	max-height: 100%;
```

Comment:
I don’t think the height matters, `*-height: 100%` is ignored when the container doesn't have a set height